app/forms/idv/phone_form.rb

```diff
@@ -31,7 +31,8 @@ def submit(params)
     attr_writer :phone
 
     def update_idv_params(phone)
-      idv_params[:phone] = phone
+      normalized_phone = phone.gsub(/([-()+ ])/, '')[1..-1]
```

Comment:
Or alternatively, what if we used a robust gem like [phonelib](https://github.com/daddyz/phonelib) to help with this normalization?

```ruby
phone = Phonelib.parse('+1 888 555 1234')
phone.national(false)
=> "8885551234"

```